cluster.go

```diff
@@ -11,6 +11,8 @@ import (
 	"time"
 )
 
+const defaultDriverName = "ScyllaDB Go Driver"
```

Comment:
do we have to have it as a const here? Couldn't it be just set in `NewCluster`? Is this const used in any other place?

Reply:
Isn't a const var generally preferred to magic numbers / strings inside code?

Reply:
true

Reply:
Just having it there as a global variable and only using in one place made me ask this question, but maybe it is the way to go

Reply:
I do not have strong opinion on that